cloud_formation/configs/api.py

```diff
@@ -441,8 +441,25 @@ def update(session, domain):
     return success
 
 def delete(session, domain):
-    names = AWSNames(domain)
-    aws.route53_delete_records(session, domain, names.endpoint)
-    aws.sqs_delete_all(session, domain)
-    aws.policy_delete_all(session, domain, '/ingest/')
-    CloudFormationConfiguration('api', domain).delete(session)
+    yes = {'yes','y', 'ye', ''}
+    no = {'no','n'}
+
+    #Check the action is desired.
+    sys.stdout.write("Are you sure you want to proceed? All data within current boss will be lost.[y/n]")
```

Comment:
May want to switch the sentence around "All data will be lost. Are you sure you want to proceed? [y/N]" so the question is next to the response

Reply:
Will do, good point